src/components/contextual/pages/vebal/LMVoting/LMVoting.vue

```diff
@@ -54,6 +54,17 @@ const votingPeriodEnd = computed<number[]>(() => {
   return formattedTime;
 });
 
+const votingPeriodLastHour = computed<number[]>(() => {
```

Comment:
This computed property is returning a boolean so we need to change the type here.
```suggestion
const votingPeriodLastHour = computed((): boolean => {
```